quesma/quesma/config/url.go

```diff
@@ -15,6 +18,12 @@ func (u *Url) UnmarshalText(text []byte) error {
 	if err != nil {
 		return err
 	}
+	if len(urlValue.Scheme) == 0 {
+		return fmt.Errorf("URL scheme (e.g. http:// or clickhouse://) is missing from the provided URL: %s", urlValue)
+	}
+	if len(urlValue.Port()) == 0 {
+		return fmt.Errorf("URL port (e.g. 8123 in 'http://localhost:8123') is missing from the provided URL: %s", urlValue)
```

Comment:
This URL parsing in Go is borked - below you can see the result of parsing `demo.host.com:9440`:
<img width="1239" alt="image" src="https://github.com/user-attachments/assets/9f7d86b1-581e-453f-b885-bba80cf6e275">
which will result in `URL port is missing`.

I don't like suggesting this, but if we want to get correct messages we should introduce our own little parsing logic (e.g. extracting `.*://` or `:[[:digit:]]`)  😬

Reply:
In order to avoid introducing our own parsing code, I changed the logic from: `len(urlValue.Scheme) == 0` to `len(urlValue.Scheme) == 0 || len(urlValue.Host) == 0` and made the error message more generic: `"Invalid URL (missing protocol and/or hostname). Expected URL in a format 'protocol://hostname:port' (e.g. 'http://localhost:8123'), but instead got: %s"`